app/Http/Controllers/Account/AcceptanceController.php

```diff
@@ -118,7 +117,21 @@ public function store(Request $request, $id)
 
             $return_msg = trans('admin/users/message.declined');
         }
+        $sig_image=Storage::get('private_uploads/signatures/'.$sig_filename);
+        $this->SignatureEulaPDF($acceptance, $sig_image );
 
         return redirect()->to('account/accept')->with('success', $return_msg);
     }
+
+    public function SignatureEulaPDF($acceptance, $sig_image)
+    {
+//        When I compact the $data I at least get back an ipsum string in the error message that is recognized as an
+//        "unknown variable". 2nd param in load view should be the array of things you wanted rendered in the pdf.
+        $data = [
+        'eula'=> $acceptance->checkoutable->getEula(),
+        'signature' => $sig_image,
+        ];
+        $pdf = PDF::loadView('account.accept.create', compact($data));
```

Comment:
Why do we need to compact this? It’s already an array,, so we should just be able to pass it?